src/PickerTop.js

```diff
@@ -0,0 +1,44 @@
+import React, {PropTypes} from 'react';
+import cx from 'classnames';
+
+export default React.createClass({
+  propTypes: {
+    fixed: PropTypes.bool,
+    handleClick: PropTypes.func,
+    nextDate: PropTypes.func,
+    previousDate: PropTypes.func,
+    title: PropTypes.string,
+    textClassNames: PropTypes.string,

```

Comment:
I just copied the name from the existing file :blush: I'll change it right away.